tests/snuba/api/endpoints/test_organization_events_v2.py

```diff
@@ -0,0 +1,152 @@
+from __future__ import absolute_import
+
+
+from datetime import timedelta
+from django.utils import timezone
+from django.core.urlresolvers import reverse
+
+from sentry.models import Group
+from sentry.testutils import APITestCase, SnubaTestCase
+
+
+class OrganizationEventsTestBase(APITestCase, SnubaTestCase):
+    def setUp(self):
+        super(OrganizationEventsTestBase, self).setUp()
+        self.min_ago = (timezone.now() - timedelta(minutes=1)).isoformat()[:19]
+        self.two_min_ago = (timezone.now() - timedelta(minutes=2)).isoformat()[:19]
+        self.url = reverse(
+            'sentry-api-0-organization-events',
+            kwargs={
+                'organization_slug': self.organization.slug,
+            }
+        )
+
+
+class OrganizationEventsV2EndpointTest(OrganizationEventsTestBase):
```

Comment:
Just a suggestion, I think it might be helpful to have a test that is representative of the views we want to show: i.e. show that it can give all data for the All Events tab: event, type, project, user (id and ip), time.  It totally looks like it can, though. Alternatively, it's ok to tests this later on as an acceptance test as well.

Reply:
Added a couple of fields to the test case